aws-lc-rs/Cargo.toml

```diff
@@ -64,7 +64,7 @@ home = "=0.5.5"
 # Pinned dependency to preserve MSRV: 1.60.0  <= rust-version < 1.65.0
 regex = "~1.9.6"
 # Pinned dependency to preserve MSRV: ??? <= rust-version < 1.65.0
-regex-automata = "~0.3.9"
+regex-automata = "~0.4.6"
```

Comment:
0.3.9 is the latest version to not require Rust 1.65.0: https://crates.io/crates/regex-automata/versions